ledger-service/http-json/src/main/scala/com/digitalasset/http/json/JsonProtocol.scala

```diff
@@ -30,6 +30,17 @@ object JsonProtocol extends DefaultJsonProtocol with ExtraFormats {
   implicit val ContractIdFormat: JsonFormat[domain.ContractId] =
     taggedJsonFormat[String, domain.ContractIdTag]
 
+  implicit val AbsoluteContractIdFormat: JsonFormat[AbsoluteContractId] =
```

Comment:
we might want to move it from `com.daml.http.json.Protocol`, this class contains JSON format definitions for `com.daml.domain`.... `AbsoluteContractId` is `com.daml.lf.value`